src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -334,6 +334,29 @@ Parameter IStreamsExecutor::Config::GetConfig(const std::string& key) const {
     return {};
 }
 
+void IStreamsExecutor::Config::UpdateHybridCustomThreads(Config& config) {
+    const auto num_cores = parallel_get_max_threads();
+    const auto num_cores_phys = getNumberOfCPUCores();
+    const auto num_big_cores_phys = getNumberOfCPUCores(true);
+    const auto num_big_cores = num_cores > num_cores_phys ? num_big_cores_phys * 2 : num_big_cores_phys;
+    const auto num_small_cores = num_cores_phys - num_big_cores_phys;
```

Comment:
Yes, it is really `num_small_cores_phys`. Because there is no small cores with hyper threads now, so I name it num_small_cores. I will rename it later.